lua/utilities.lua

```diff
@@ -407,6 +407,10 @@ function utilities.reply_markup_from_text(text)
 	return reply_markup, clean_text
 end
 
+function utilities.reply_from_msg(msg)
```

Comment:
It's not very clear why this function is needed.

Reply:
Well, I could simply use msg.reply instead

Reply:
Dew it!